### ISSUE TITLE
Installation fails: Unknown sub-command: 'cp'

### ISSUE THREAD

**Issue by adventmedia** [xdissent/ievms#262]
Possibly an issue with Virtualbox 5.0?
Running the script to install IE 9, 10 and 11; at some point the process aborts with the error:
Unknown sub-command: 'cp'
Previous statement is usage for VBoxManage guestcontrol.
IE9 and IE10 are installed at this point, but IE10 is in a running state.

**Comment by nbudin** [comment]
Same here.  I've cloned the ievms repo locally and am working on trying to get it working with VB 5.0, with the intent to make a pull request, but I'm still struggling a bit with some of the later steps in the process.

**Comment by xdissent** [comment]
Yeah, they removed and renamed some VBoxManage sub-commands in VBox 5 (broke boot2docker as well). I'm on the case tho!

**Comment by nbudin** [comment]
Thanks @xdissent!  :+1:

**Comment by xdissent** [comment]
Wow, I think they really boned the 5.0 release. Specifically, I'm pretty sure they 100% broke copying files from host to guest, which is crucial. I'm trying to put a finer point on the specific issues and I'll file bug reports tomorrow.

**Comment by arthurmuchir** [comment]
+1 with sub-command 'exec'

**Comment by peterjwest** [comment]
I'm getting the same issue with 'exec'

**Comment by aircrew** [comment]
Same here with sub-command 'exec'

**Comment by NodeReaver** [comment]
I had the same issue with sub-command 'exec' when I installed VirtualBox 5.
Using the older VirtualBox-4.3.10-93012-OSX image linked by this article fixed the problem http://osxdaily.com/2011/09/04/internet-explorer-for-mac-ie7-ie8-ie-9-free/

**Comment by peterjwest** [comment]
VirtualBox 5 has changed it's API, so we need an updated script to use it.

**Comment by xdissent** [comment]
Yeah, the `exec` syntax change is weird and annoying, but the bigger issue (since we can't work around it) is that the `copy` commands are broken altogether. For one, there's an obvious argument parsing bug (classic off-by-one), but there is also some deeper bug that totally prevents the copy operation... I just haven't been able to find the cause yet. Stick with 4.x.x for now if you can.

**Comment by n3r0-ch** [comment]
Same here with sub-command 'exec'

**Comment by vvo** [comment]
Good luck @xdissent I love ievms, will switch to saucelabs manual testing for now. Thx

**Comment by kimbo6365** [comment]
+1 for installing VirtualBox 4.3.x. Got me up and running.

**Comment by n0nag0n** [comment]
+1 on the exec issue :(

**Comment by xdissent** [comment]
I pushed a potential fix in the `vbox-5` branch. Give it a try:

```console
$ curl -s https://raw.githubusercontent.com/xdissent/ievms/vbox-5.0/ievms.sh | bash
```

**Comment by xdissent** [comment]
Fixed in `v0.3.0`
